rules/defaults.yaml

```diff
@@ -278,13 +278,15 @@
 		DeathTypes:
 			ExplosionDeath: 1
 			BulletDeath: 2
+			ElectroDeath: 6
 	WithDeathAnimation@effect:
 		DeathSequencePalette: effect
 		DeathPaletteIsPlayerPalette: False
 		DeathTypes:
 			RadiationDeath: 4
-			FlameDeath: 5
-			ElectroDeath: 6
```

Comment:
Ah, we have an SHP override in our bits folder which is indeed converted to unittem - removing that file should be the fix.